docs/installation.rst

```diff
@@ -1,5 +1,145 @@
+2. .. _Playbook install docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/installing-an-identity-management-replica-using-an-ansible-playbook_installing-identity-management
+
+
+Install ansible and ansible-freeipa rpms
+
+::
+    subscription-manager repos --enable ansible-2.8-for-rhel-8-x86_64-rpms
+    yum install ansible
+    yum install ansible-freeipa
+
+
+The location where the ansible roles are installed:
+
+::
+    [root@server]# ls -1 /usr/share/ansible/roles/
+    ipaclient
+    ipareplica
+    ipaserver
+
+
+The location where the documentation is once installed:
+
+::
+    [root@server]# ls -1 /usr/share/doc/ansible-freeipa/
+    playbooks
+    README-client.md
+    README.md
+    README-replica.md
+    README-server.md
+    README-topology.md
+
+
+The location where the ansible playbooks are once installed:
+
+::
+    [root@server]# ls -1 /usr/share/doc/ansible-freeipa/playbooks/
+    install-client.yml
+    install-cluster.yml
+    install-replica.yml
+    install-server.yml
+    uninstall-client.yml
+    uninstall-cluster.yml
+    uninstall-replica.yml
+    uninstall-server.yml
+
+
+A sample playbook making use of the role to install the server:
+
+::
+    [root@server]# cat install_server.yaml
+    ---
+    - name: Playbook to configure IPA server
+      hosts: ipaserver
+      become: true
+
+      roles:
+      - role: ipaserver
+        state: present
+
+    [root@server]# ansible-playbook -i inventory install_server.yaml
+
+
+
+A sample playbook making use of the install a replica role:
+
+::
+    [root@server]# cat install_replica.yaml
+    ---
+    - name: Playbook to configure IPA replicas
+      hosts: ipareplicas
+      become: true
+
+      roles:
+      - role: ipareplica
+        state: present
+
+    [root@server]# ansible-playbook -i inventory install_replica.yaml
+
+
+===================
+Noggin Installation
+===================
+
+Clone the Fedora infrastructure ansible repo:
+
+::
+    [root@server]# git clone https://pagure.io/fedora-infra/ansible.git
+
+
+
+Example inventory:
+
+::
+    [root@server]# cd ansible
+    [root@server]# cat inventory
+    [os_masters_stg]
+    os-master01.stg.iad2.fedoraproject.org
+    os-master02.stg.iad2.fedoraproject.org
+    os-master03.stg.iad2.fedoraproject.org
+
+    [os_infra_nodes_stg]
+    os-node01.stg.iad2.fedoraproject.org
+    os-node02.stg.iad2.fedoraproject.org
+    os-node03.stg.iad2.fedoraproject.org
+    os-node04.stg.iad2.fedoraproject.org
+    os-node05.stg.iad2.fedoraproject.org
+
+    [os_nodes_stg:children]
+    os_infra_nodes_stg
+
+
+    [os_control]
+    os-control01.iad2.fedoraproject.org
+
+    [os_stg:children]
+    os_nodes_stg
+    os_masters_stg
+    os_control_stg
+
+
+Run the installation playbook:
+
+::
+    [root@server]# ansible-playbook -i inventory/inventory playbooks/openshift-apps/noggin.yml
```

Comment:
This is definitely going to need some further configuration with secrets etcetera, because clones of `ansible.git` won't have that outside of the Fedora batcave.

Reply:
I think I've addressed this part of the requested changes, I've added the ansible variables which are used within the playbook.